### ISSUE TITLE
@astrouxds/react source maps fail to load because src folder is missing in npm package

### ISSUE THREAD

**Issue by EzraBrooks** [RocketCommunicationsInc/astro#215]
When compiling a React TypeScript app that uses AstroUXDS: 
```
WARNING in ./node_modules/@astrouxds/react/dist/react-component-lib/createOverlayComponent.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/ezra/Development/PickNik/moveit_studio/moveit_studio_web/node_modules/@astrouxds/react/src/react-component-lib/createOverlayComponent.tsx' file: Error: ENOENT: no such file or directory, open '/Users/ezra/Development/PickNik/moveit_studio/moveit_studio_web/node_modules/@astrouxds/react/src/react-component-lib/createOverlayComponent.tsx'
```

`src` folder doesn't exist:
```
~/D/P/m/moveit_studio_web (main|✚2…) $ ls /Users/ezra/Development/PickNik/moveit_studio/moveit_studio_web/node_modules/@astrouxds/react/src/
ls: /Users/ezra/Development/PickNik/moveit_studio/moveit_studio_web/node_modules/@astrouxds/react/src/: No such file or directory
```

This produces massive amounts of compiler warnings, to the point of it making it hard to read my build logs 😦

**Comment by EzraBrooks** [comment]
oops, saw you just fixed this recently

**Comment by markacianfrani** [comment]
@EzraBrooks yep we'll have this published on thursday in our 6.1 release
